errors.go

```diff
@@ -66,7 +66,7 @@ var (
 	// to determine if it is safe to delete. "conflict" followed by newline is used to
 	// preserve go-tfe version compatibility with the error constructed at runtime before it was
 	// defined here.
-	ErrWorkspaceStillProcessing = errors.New("conflict\nworkspace is still being processed to discover resources")
+	ErrWorkspaceStillProcessing = errors.New("conflict\nlatest workspace state is still being processed, and attempting to discover resources")
```

Comment:
I like the addition of "latest workspace state" but don't like the addition of ", and attempting" because that is the purpose of processing and not a separate operation. What do you think about fully duplicating the API error?

```suggestion
	ErrWorkspaceStillProcessing = errors.New("conflict\nLatest workspace state is being processed to discover resources, please try again later")
```